script/firebaseScriptLogWork.js

```diff
@@ -87,6 +87,7 @@ function createNewTask(title,des,priority,difficulty,creator,assignee,estimatedT
     Status: status,
     Category: category,
     LogWorkTotalTime: loggedWorkTotalTime,
+    Date: dates,
```

Comment:
loggingDate will sound much better than dates

Reply:
Okay, I will change that.